test/controllers/TransferControllerTest.scala

```diff
@@ -390,12 +389,38 @@ class TransferControllerTest extends ControllerBaseSpec {
 
   "finished" should {
     "return success" when {
-      "A notification record is returned" in {
+      "make sure method remove never was called before" in {
+        verify(mockCachingService, times(0)).remove()(any(), any())
```

Comment:
You arent calling anything for the verify to work here?

Reply:
I split one test in 3 test(3 steps just to be readable)

verify method is checking as method remove was called 0 times

and next step I call controller().finished() which actual contain inside .remove() method and then I check as this method was called exactly once